fasm2bels/database/connection_database_cache.py

```diff
@@ -1,3 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
```

Comment:
Missing newline here.

Reply:
Actually, yapf complains if a new line is added there. Not sure how to fix it.

I think a possible way would be to move the comment below the imports, or change some yapf configs?

Reply:
Move the comment below the imports I think?